metrics/cli/docs/05_user_manual_steps.md

```diff
@@ -0,0 +1,163 @@
+
+During the deployment process, we should display the following prompt explaining how to enable the Firestore analytics:
+
+```text
+To enable Firebase analytics for the currently deploying project, follow the next steps:
+
+1. Follow the link and click an "Enable Google Analytics" button: https://console.firebase.google.com/project/project_id/settings/integrations/analytics
+
+Once you are done, press any key to continue:
+```
+
+### Firebase auth configuration
+
+To simplify the Firebase auth configuration, we are going to provide instructions on how to configure it for the Metrics application during the deployment process. 
+
+These instructions will contain information on how to configure the following auth providers: 
+- [Google](https://github.com/platform-platform/monorepo/blob/master/docs/09_firebase_deployment.md#firebase-google-sign-in-configuration)
+- [Email and Password](https://github.com/platform-platform/monorepo/blob/master/docs/09_firebase_deployment.md#firebase-email-and-password-sign-in-configuration)
+
+Consider the following prompt instructions helping to configure the Firebase Auth:
+
+```text
+To enable the Firebase Auth for the Metrics Web application, consider the following steps:
+
+1. Follow the link and click a "Get started" button: https://console.firebase.google.com/project/project_id/authentication
+2. Navigate to the 'Sign-in method' tab on the Firebase auth page or use the following link: https://console.firebase.google.com/project/project_id/authentication/providers
+3. Enable an Email/Password provider by clicking on provider name and toggling the `Enable` switch.
+4. Enable a Google provider by clicking on provider name and toggling the `Enable` switch.
+5. On the 'Google' provider popup, open the 'Web SDK configuration' tab, copy the Web client ID and paste it to the console
+
+Paste your Web client Id here:
+```
+
+### Sentry Configuration
+
+Since the Metrics Web application uses Sentry, we should provide detailed instruction to help a user configure a Sentry release during the Metrics application deployment process.
+Let's review prompts instructions that will help configure a new Sentry release.
+
+The first prompt for authorizing to the Sentry and retrieving an `Organization Slug`:
+```text
+To enable Sentry for the Metrics Web application, consider the following steps:
+
+The following steps help to find an `Organization Slug` for the Sentry account:
+
+1. Visit the following link and authorize: https://sentry.io
+2. Navigate to the Sentry's `Settings` tab
+3. Navigate to the `General Settings` tab and copy an `Organization Slug`
+
+Paste the `Organization Slug` here:
+```
+
+The prompt for retrieving a `Project Slug`:
+```text
+The following steps help to find a `Project Slug` for the Sentry account:
+
+1. Navigate to the Sentry's `Settings` tab
+2. Navigate tp the `Projects` tab and copy a required `Project Slug`
```

Comment:
```suggestion
2. Navigate to the `Projects` tab and copy a required `Project Slug`
```
Also, would it be great to give a link to users to simplify our steps? I think we can create a link since we already have an `organization slug`. What do you think?